docs/quickstart.md

```diff
@@ -85,7 +85,12 @@ ansible-playbook install.yml -u <managed_node_username> --ask-become-pass
 
 Storage
 -------
-To set up and configure Amazon S3 or Azure storage to use with Pulp, follow the storage configuration instructions in [our docs](https://docs.pulpproject.org/pulpcore/installation/storage.html). View the [example playbooks](https://github.com/pulp/pulp_installer/tree/master/playbooks) and note the `pulp_settings` variables for your chosen storage type. Add an entry for each of the variables for your deployment.
+
+By default, Pulp uses the local filesystem for storage. If you want to use another storage backend such as Amazon Simple Storage Service (S3) or Azure, you’ll need to configure Pulp. To add an Amazon S3 or Azure backend for Pulp using the installer, you must set the necessary variables for your cloud storage in the `pulp_settings` role.
```

Comment:
`pulp_settings` is not a role, it is a variable needed at `pulp_common` role, we usually set this variable on the playbook

Reply:
Hope this is better. If I understand correctly, in the playbook, you declare the variables you want to customize for your deployment, but to figure out which variables, you need to consult the role documentation? Perhaps I should spend the day of learning doing some Ansible training!

Reply:
Yes! You got it! It is exactly like this.